js/controller.js

```diff
@@ -857,7 +868,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         // If the core tells us that it will autoplay then we just display the loading
         // spinner, otherwise we need to render the big play button.
         if (params.willAutoplay) {
-          this.state.screenToShow = CONSTANTS.SCREEN.LOADING_SCREEN;
+          this.state.screenToShow = CONSTANTS.SCREEN.START_LOADING_SCREEN;
```

Comment:
What is the difference between LOADING_SCREEN and START_LOADING_SCREEN? What is START_LOADING_SCREEN used for?

Reply:
Black screen + spinner = LOADING_SCREEN
Thumbnail (promo image)+spinner = START_LOADING_SCREEN

There is no transition between these screens. We show one or another.